log-collector/src/main/scala/io/phdata/pulse/logcollector/SolrCloudStreams.scala

```diff
@@ -60,9 +71,9 @@ class SolrCloudStreams(solrService: SolrService) extends LazyLogging {
    */
   val groupedInsert: RunnableGraph[ActorRef] = {
     Source
-      .actorRef[(String, LogEvent)](Int.MaxValue, OverflowStrategy.dropNew)
+      .actorRef[(String, LogEvent)](maxBuffersize, OverflowStrategy.dropNew)
       .groupBy(MAX_SUBSTREAMS, x => x._1) // group by the application name
-      .groupedWithin(GROUP_SIZE, GROUP_MAX_TIME) // group 1000 records or within one second, whichever comes first
+      .groupedWithin(solrBatchSize, batchFlushDuration) // group 1000 records or within one second, whichever comes first
```

Comment:
I recommend removing the reference to hard values from the comment, or indicate they're the defaults.

Reply:
Good, done.